static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -126,19 +126,18 @@ const isCompatibleWithLiveBlogEpic = (page: Object): boolean =>
 const pageShouldHideReaderRevenue = () =>
     config.get('page.shouldHideReaderRevenue');
 
+const isCompatibleUser = (onlyShowToExistingSupporters: boolean) =>
+    onlyShowToExistingSupporters ? userIsSupporter() : !userIsSupporter();
```

Comment:
`onlyShowToExistingSupporters` suggests that if it's false it shows to everyone, perhaps the name could be improved? Not sure if an enum is lightweight enough to be usable here as boolean is prone to error

Reply:
Yeah me & Tom have been round the houses with this one a lot and never found a nice way of doing it with a boolean. So I actually really like the idea of something like

`type UserCohort = 'OnlyExistingSupporters' | 'OnlyNonSupporters' | 'Everyone';`

Reply:
or `TargetedUserCohort`
or `TargetsUsers`

Reply:
I guess to do that properly we would set it that way in the test types (epic & banner) too. And then maybe eventually in the sheet, though we could leave that for the moment

Reply:
I ❤️ it. Have a look and see what you think @johnduffell @joelochlann